packages/sdk/identity/src/odis/matchmaking.ts

```diff
@@ -42,6 +45,9 @@ export async function getContactMatches(
   if (sessionID) {
     body.sessionID = sessionID
   }
+  if (dekSigner) {
+    body.signedUserPhoneNumber = signWithDEK(selfPhoneNumObfuscated, dekSigner)
```

Comment:
TODO: Create documentation recommending that developers supply an EncryptionKeySigner when calling getContactMatches()

Reply:
TODO: Create ticket for Valora to start supplying an EncryptionKeySigner when calling getContactMatches()